OpenDreamRuntime/Objects/Types/DreamObjectWorld.cs

```diff
@@ -32,14 +39,27 @@ public sealed class DreamObjectWorld : DreamObject {
         set => _gameTiming.TickRate = (byte)value;
     }
 
+    private DreamList? _paramsCache;
+
+    private DreamList Params {
```

Comment:
In BYOND this can be assigned to something other than a list, so this can just be made a DreamValue. I don't think a setter is necessary since there's no point in updating the CVar, and the getter can be replaced by initializing the value in the constructor.